src/domain/individualHarbor/forms/berthForm/utils/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { INDIVIDUAL_BERTH } from '../__generated__/INDIVIDUAL_BERTH';
+import { Berth } from '../types';
+
+export const getBerth = (berthData: INDIVIDUAL_BERTH | undefined): Berth => {
+  if (!berthData || !berthData.berth) return {};
```

Comment:
Should we return `null` or `undefined` instead to be more explicit?

Reply:
Done.